virtcontainers/sandbox.go

```diff
@@ -1951,6 +1955,12 @@ func (s *Sandbox) updateResources() error {
 func (s *Sandbox) calculateSandboxMemory() int64 {
 	memorySandbox := int64(0)
 	for _, c := range s.config.Containers {
+		// Do not hot add again non-running containers resources
+		if cont, ok := s.containers[c.ID]; ok && cont.state.State == types.StateStopped {
```

Comment:
Then you need to recalculate resources in `s.StartContainer()`, right?

Reply:
good point!, let me do it and run the tests